packages/react-native-sdk/src/hooks/useAndroidKeepCallAliveEffect.ts

```diff
@@ -11,12 +16,25 @@ import {
 const notifeeLib = getNotifeeLibNoThrowForKeepCallAlive();
 
 function setForegroundService() {
-  notifeeLib?.default.registerForegroundService(() => {
-    return new Promise(() => {
-      const logger = getLogger(['setForegroundService method']);
-      logger('info', 'Foreground service running for call in progress');
-    });
-  });
+  if (!notifeeLib) return;
+  NativeModules.StreamVideoReactNative.isCallAliveConfigured().then(
+    (isConfigured: boolean) => {
+      if (!isConfigured) {
+        const logger = getLogger(['setForegroundService method']);
+        logger(
+          'info',
+          'Skipping KeepCallAlive foreground service setup as it is not configured properly'
```

Comment:
```suggestion
          'KeepCallAlive is not configured. Skipping foreground service setup.'
```

Maybe this is intentional. `properly` sounds like we want customers to always enable it.

Reply:
changed